### ISSUE TITLE
MaterialTimePicker setValue after clear not working

### ISSUE THREAD

**Issue by 8vale8** [GwtMaterialDesign/gwt-material-addins#139]
The clear method of MaterialTimePicker does not keep up to date the private field "time".
So after calling clear() it is impossible to set a new value programmatically with setValue because it performs the following check:

`if(this.time != null) {
            if(this.time.equals(time)) {
                return;
            }
        }`

**Comment by kevzlou7979** [comment]
Ok will check this out :) Thanks for posting :)

**Comment by kevzlou7979** [comment]
Can you post the code so that I can reproduce it, I cant reproduce on my side about the behaviour.

Here's the code I've used, the time picker always set after clearing :)
![test2](https://cloud.githubusercontent.com/assets/3138071/18532997/66c32d42-7b12-11e6-9d0d-eeb8436ca259.gif)

```
        tpClear.setPlaceholder("Time");
        btnClear.setText("Clear");
        btnClear.addClickHandler(clickEvent -> {
            tpClear.clear();
        });

        btnSetValue.setText("Set Value");
        btnSetValue.addClickHandler(clickEvent -> {
            tpClear.setValue(new Date());
        });
```

**Comment by BenDol** [comment]
What version are you testing on? Need to identify the version that has the issue.

**Comment by kevzlou7979** [comment]
I have tested on 2.0 Snapshot specially mk_api_work branch

**Comment by kevzlou7979** [comment]
https://gwtmaterialdesign.github.io/gwt-material-demo-errai/1.0-SNAPSHOT/#timepicker

See here, 2.0 Snapshot ive created demo on last example

**Comment by BenDol** [comment]
The issue does exist, already fixed it in 2.0

**Comment by kevzlou7979** [comment]
Ok thanks for the confirmation
